components/card/ProfilePersonCard.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <div
+    class="profile-person-card"
+    :style="profileCardStyleAsPersonType"
+    itemscope
+    itemtype="https://schema.org/Person"
+  >
+    <div class="row">
+      <div class="col-12 col-lg-3 d-flex justify-content-center">
+        <img
+          v-if="person.avatar.length > 0"
+          :src="person.avatar"
+          class="profile-person-card__avatar"
```

Comment:
Convension olarak bem kullanmadik diger yerlerde, burada bem'e kaymisiz sanki biraz

Reply:
person card avatar

Reply:
belli bi metodoloji yok diye kafama göre yazdım :D bem'in bi eksisi olacağını düşünmüyorum